find-gadgets.py

```diff
@@ -276,7 +277,11 @@ def main(args):
     print_useful_regex(args.output, args.arch)
 
 
-if __name__ == "__main__":
+if __name__ == "__main__":    
+    #Fix issue with Ropper in macOS -> AttributeError: 'Ropper' object has no attribute '__gatherGadgetsByEndings'
+    multiprocessing.freeze_support()
+    multiprocessing.set_start_method('fork')
```

Comment:
just like you suggested, let's gate this with an os check

```suggestion
    if platform.system() == "Darwin":
        multiprocessing.set_start_method('fork')
```

https://docs.python.org/3/library/platform.html#platform.system

Reply:
Let's not forget to add
import platform